src/gui/DetailsWidget.cpp

```diff
@@ -338,3 +359,28 @@ void DetailsWidget::updateTabIndex(int index)
         m_selectedTabEntry = index;
     }
 }
+
+void DetailsWidget::openAttachment(const QModelIndex& index)
```

Comment:
Can't we use the `openAttachment` from the EditEntryWidget class? Or alternatively declaring it static in a different class and point both of them to the static one? This is just repeated code and I would like to leave the detailsWidget light

Reply:
Agree with you. I'll fix it tonight.

Reply:
@TheZ3ro perhaps it is better to create a separate widget to display/edit the attachments. With the functionality of adding, deleting and opening files (which can be turned off in the details view).

Reply:
It's fine opening files from the details view, it's the extra code that I don't really like.
Making a separate widget and import it will be very nice and convenient